public/data/en/states.json

```diff
@@ -8,7 +8,7 @@
     "reg_type": "online",
     "accepts_nvrf": "1",
     "mailing_address_inst": "<p>Office of the Secretary of State  <br />P.O. Box 5616 <br />Montgomery, AL 36103-5616</p>",
-    "reg_eligibility_desc": "<p>To register in Alabama you must:  </p><ul><li>Be a citizen of the United States  </li><li>Be a resident of Alabama and your county at the time of registration</li><li>Be 18 years old before any election</li><li>Not have been convicted of a felony involving moral turpitude (or have had your civil and political rights restored). The list of moral turpitude felonies is available on the Secretary of State web site at: <a href=\"sos.alabama.gov/mtfelonies\">sos.alabama.gov/mtfelonies</a>  </li><li>Not currently be declared mentally incompetent through a competency hearing  </li><li>Swear or affirm to “support and defend the Constitution of the U.S. and the State of Alabama and further disavow any belief or affiliation with any group which advocates the overthrow of the governments of the U.S. or the State of Alabama by unlawful means and that the information contained herein is true, so help me God”  <br /> </li></ul>",
+    "reg_eligibility_desc": "<p>To register in Alabama you must:  </p><ul><li>Be a citizen of the United States  </li><li>Be a resident of Alabama and your county at the time of registration</li><li>Be 18 years old before any election</li><li>Not have been convicted of a felony involving moral turpitude (or have had your civil and political rights restored). The <a href=\"https://alisondb.legislature.state.al.us/alison/CodeOfAlabama/1975/17-3-30.1.htm\">list of moral turpitude felonies</a> is available on the Secretary of State web site</li><li>Not currently be declared mentally incompetent through a competency hearing  </li><li>Swear or affirm to “support and defend the Constitution of the U.S. and the State of Alabama and further disavow any belief or affiliation with any group which advocates the overthrow of the governments of the U.S. or the State of Alabama by unlawful means and that the information contained herein is true, so help me God”  <br /> </li></ul>",
```

Comment:
should we make this to open in a new tab like other external links?

Reply:
@clmedders Yes, good catch. I just pushed a change.